### PR TITLE
RobotFrameworkAssignment 

### PR DESCRIPTION
**Pull request by santoshilondhe**
Robot Framework Assignments added
Ready for your review!

### CODE REVIEW

**Review by komalkapde — APPROVED** [review]

**Review by shriram3672 — APPROVED** [review]
LGTM

File: RobotFramework/Tests/SouceDemo.robot [hunk 1]
```diff
@@ -0,0 +1,17 @@
+*** Settings ***
+Documentation       This Test file contains all TC's related to SouceDemo
+Resource        ../Resources/SouceDemo_Keywords.robot
+
+*** Variables ***
+
```
Comment (plaad-mdsol) [inline RobotFramework/Tests/SouceDemo.robot:6]: Remove variable section if not req

File: RobotFramework/Configuration/Uat_Env.robot [hunk 2]
```diff
@@ -0,0 +1,5 @@
+*** Variables ***
+${URL}          https://uat.goibibo.com/
+${URL}          https://uat.soucedemo.com/
+${BROWSER}      Chrome
+${ENV}          Uat
```
Comment (pcube0608) [inline RobotFramework/Configuration/Uat_Env.robot]: ENV variable we can remove from here and parse as commandline argument
Please remove and commit from all env files

File: RobotFramework/Tests/SouceDemo.robot [hunk 3]
```diff
@@ -0,0 +1,17 @@
+*** Settings ***
+Documentation       This Test file contains all TC's related to SouceDemo
+Resource        ../Resources/SouceDemo_Keywords.robot
+
+*** Variables ***
```
Comment (pcube0608) [inline RobotFramework/Tests/SouceDemo.robot]: Remove variable section

File: RobotFramework/Resources/Goibibo_Keywords.robot [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (pcube0608) [inline RobotFramework/Resources/Goibibo_Keywords.robot]: Pls remove all blank line from end of file

File: RobotFramework/Resources/SouceDemo_Keywords.robot [hunk 5]
```diff
@@ -0,0 +1,35 @@
+*** Settings ***
+Documentation    This Resource file contains implementaion of SouceDemo
+Library          SeleniumLibrary
+Resource         ../Configuration/${ENV}_Env.robot
+
+
+*** Keywords ***
+Login to the SouceDemo website
```
Comment (pcube0608) [inline RobotFramework/Resources/SouceDemo_Keywords.robot]: This line can be removed

**Review by pcube0608 — CHANGES_REQUESTED** [review]

**Review by pcube0608 — APPROVED** [review]

**Pull request merged**